animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -137,6 +137,8 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         col.prop(self, "mode", text = "")
         if self.mode in ("LINEAR", "GRID"):
             col.prop(self, "distanceMode", text = "")
+            if self.mode == "LINEAR":
+                layout.prop(self, "centerAlongX", text = "Center Linear", toggle = True)
```

Comment:
If we will be sharing those properties between modes, make the property description more general above.

Reply:
![linear mode](https://user-images.githubusercontent.com/64368228/104112766-8b432800-532d-11eb-906d-ea6d63058310.gif)

is this OK?

Reply:
looks good to me at least. Thanks for development.